robotidy/transformers.py

```diff
@@ -0,0 +1,38 @@
+import inspect
+import sys
+
+from robot.parsing import ModelTransformer
+
+
+def load_transformers(modes):
+    """ Dynamically load all classess from this file with attribute `name` defined in modes """
+    transfomers = []
+    classes = inspect.getmembers(sys.modules[__name__], inspect.isclass)
+    for transfomer_class in classes:
+        transfomer = transfomer_class[1]()
+        if getattr(transfomer, 'name', None) in modes:
```

Comment:
`isinstance(transformer, ModelTransformer)` ought to work as well. Classes have `__name__` that can be used if a name is needed.

Reply:
I've updated the code to use `__name__` instead of name class attribute. Now we need to use class name to use transformer:

```
robotidy --mode AnotherTransformer
```